src/utils/advancedSearchUtils.ts

```diff
@@ -0,0 +1,38 @@
+import type { SearchParams, SearchFormInputField } from "../types/searchTypes"
+import { searchAggregations } from "../config/aggregations"
+
+export const textInputFields: SearchFormInputField[] = [
+  { name: "q", key: "searchKeywords", label: "Keywords" },
+  { name: "title", key: "title", label: "Title" },
+  { name: "contributor", key: "contributor", label: "Author" },
+  { name: "subject", key: "subject", label: "Subject" },
+]
+
+export const initialSearchFormState: SearchParams = {
+  searchKeywords: "",
+  title: "",
+  contributor: "",
+  subject: "",
+  selectedFilters: {
+    language: "",
```

Comment:
There's an issue selecting a language because there's no key:
<img width="1165" alt="Screen Shot 2023-09-20 at 11 34 23 AM" src="https://github.com/NYPL/research-catalog/assets/1280564/3d5e1f4b-5b7f-4382-b5d3-411b03ed8fe1">

Reply:
Good catch, fixed and improved the comment explaining this key lookup.